### ISSUE TITLE
WCAG 3.3.1 (Level A) / 4.1.3 (Level AA): Field input error is not announced in full - Missing `aria-atomic` on assertive live region

### ISSUE THREAD

**Issue by bspeare** [ckeditor/ckeditor5#19888]
## 📝 Reproduction steps 

1. Run Siteimprove [plugin](https://chromewebstore.google.com/detail/siteimprove-accessibility/djcglbmbegflehmbfleechkjhmedcopn?hl=en) on https://ckeditor.com/docs/ckeditor5/latest/examples/builds/classic-editor.html. 
2. Look for  "Field input error is not announced in full" `A` and `AA` in the Issues list. 
3. Highlight the code in the developer tools. 

### ✔️ Expected result

The assertive live region rendered should include `aria-atomic="true"`:

### ❌ Actual result

The live region is rendered without `aria-atomic`

### ❓ Possible solution

Include `aria-atomic="true"` in https://github.com/ckeditor/ckeditor5/blob/master/packages/ckeditor5-ui/src/arialiveannouncer.ts.

## 📃 Other details
**Flagged element in DOM**:
```
html
<div aria-live="assertive" aria-relevant="additions">
  <ul class="ck ck-aria-live-region-list"></ul>
</div>
```
---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.
